CONTRIBUTING.md

```diff
@@ -37,3 +37,37 @@ Contributions made by corporations are covered by a different agreement than the
  
 3) Add or change sample Gist IDs to the [index file](https://github.com/dart-lang/dart-pad/blob/master/web/index.html#L54),
    and submit a PR for review.
+
+## How to run DartPad locally
+
+DartPad requires npm v5 or higher and vulcanize to be installed.
```

Comment:
It seems like node v6 is required – not greater. At least on my machine...

Reply:
Ah.  My Linux box seems to work ok with npm v5.. maybe it depends.